packages/grid/src/vaadin-grid-data-provider-mixin.d.ts

```diff
@@ -138,4 +140,21 @@ export declare class DataProviderMixinClass<TItem> {
    * Clears the cached pages and reloads data from dataprovider when needed.
    */
   clearCache(): void;
+
+  /**
+   * Scroll to a specific row index in the virtual list. Note that the row index is
+   * not always the same for any particular item. For example, sorting or filtering
+   * items can affect the row index related to an item.
+   *
+   * The `indexes` parameter can be either a single number or multiple numbers.
+   * The grid will first try to scroll to the item at the first index on the top level.
+   * In case the item at the first index is expanded, the grid will then try scroll to the
+   * item at the second index within the children of the expanded first item, and so on.
+   * Each given index points to a child of the item at the previous index.
+   *
+   * Using `Infinity` as an index will point to the last item on the level.
+   *
+   * @param indexes Row indexes to scroll to
```

Comment:
No need to use `@param` in `.d.ts` files. Not a blocker though.

Reply:
Done